### DISCUSSION TITLE
Delete `solve`?

### DISCUSSION THREAD

**Discussion by mlutze** [flix/flix#8529] [Q&A] [answered]
What is the purpose of the `solve` keyword? It seems that because we interact with Datalog via queries, executing `solve` is never observable.

**Comment by magnus-madsen** [comment, marked as answer]
I wanted to, but I found some applications. For example, if you want to extract two relations without computing the fixpoint twice.

**Comment by JonathanStarup** [comment]
But what if we rename it to something more specific? I feel like solve is a prime keyword to reserve for a niche operation

**Comment by magnus-madsen** [comment]
> But what if we rename it to something more specific? I feel like solve is a prime keyword to reserve for a niche operation

Suggestions welcome.
